drift-transport-netty/src/main/java/com/facebook/drift/transport/netty/client/DriftNettyConnectionFactoryConfig.java

```diff
@@ -40,6 +40,7 @@
 
     private Duration sslContextRefreshTime = new Duration(1, MINUTES);
     private HostAndPort socksProxy;
+    private boolean nativeTransportEnabled = true;
```

Comment:
I would conservatively set it to `false` by default

Reply:
done